backend/routes/courses.js

```diff
@@ -298,4 +298,45 @@ router.post("/updateCourse", async (req, res, next) => {
   }
 });
 
+/**
+ *
+ * @api {post} /assignCourseToCurriculum
+ * @apiDescription assign a course to a curriculum (core, techComp, complementaries)
+ * @apiParam (body) {string} curriculumType, {string} courseCode, {string} curriculum
+ * @apiExample {curl} Example usage:
+ *	curl -X POST \
+ *  -H 'Content-Type: application/json' \
+ *  -d '{"curriculumType": "core", "courseCode": "ECSE 428", "curriculum": "ee"}' \
```

Comment:
It might be beneficial in terms of overall speed to process a course with a batch of curriculums for a single class instead of one at a time. Maybe something like this

```
{
  "course": "ECSE 428",
  "curriculums": [
    {"type": "core", "curriculum": "ee"},
    {"type": "core", "curriculum": "se-minor"}
  ]
}
```

This would need significant change and would be better left for next sprint

Reply:
I thought about it actually, but I'm not sure what the front-end would be like.  For example if they just using buttons to assign a course to one curriculum and submit, what I have now is sufficient.  I suggest to keep it like this, and if what you suggested is needed by the front-end, we definitely can improve it.

Reply:
Since we're pressed for time I agree to leave it like this for now as well. Should be modified to what steven suggested in the future though

Reply:
That's definitely a good improvement and I'm open to it. However, I would suggest to hear from front-end first and see how they are designing the UI to assign a course.
If they are assigning a course to a list of curriculum, we could go with what Steven said, but if they decided to assign it one by one, then we can keep this.